crates/pint-deploy/src/main.rs

```diff
@@ -62,12 +63,23 @@ async fn run(args: Args) -> anyhow::Result<()> {
         print_deploying(&name, &contract);
         let output = builder_client
             .register_contract(
-                big_bang.contract_registry,
-                big_bang.program_registry,
+                &big_bang.contract_registry,
+                &big_bang.program_registry,
                 &contract,
                 &programs,
             )
             .await?;
+
+        join_all(programs.iter().map(|p| {
```

Comment:
Unfortunately, as unintuitive as it seems, I _think_ this will still execute each of these futures synchronously. This is totally fine for now, just maybe worth opening an issue to remove `futures` crate and switch to using `tokio` `JoinSet` I think it's called as follow-up.

Reply:
ahh, I see.
I will proceed, but opening an issue to track this.

Reply:
ref: #152